python/pylibcudf/pylibcudf/tests/test_nvtext_replace.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import pyarrow as pa
+import pylibcudf as plc
+import pytest
+from utils import assert_column_eq
+
+
+@pytest.fixture(scope="module")
+def input_col():
+    arr = ["the quick", "brown fox", "jumps*over the", "lazy dog"]
+    return pa.array(arr)
+
+
+@pytest.mark.parametrize("delim", ["*"])
+def test_replace_tokens(input_col, delim):
+    targets = pa.array(["quick", "fox", "jumps", "dog"])
+    replacements = pa.array(["slow", "cat", "looked", "rat"])
+    result = plc.nvtext.replace.replace_tokens(
+        plc.interop.from_arrow(input_col),
+        plc.interop.from_arrow(targets),
+        plc.interop.from_arrow(replacements),
+        plc.interop.from_arrow(pa.scalar(delim)) if delim else None,
```

Comment:
Did you mean to also test the None case here? If not, then perhaps remove this block. If so, the parametrization above is missing this case, and since the test below has it too maybe convert this to a parametrized fixture.